forte/data/data_store.py

```diff
@@ -1079,6 +1124,48 @@ def add_image_annotation_raw(
                 return tid_search_result
         return self._add_entry_raw(AudioAnnotation, type_name, entry)
 
+    def add_payload_raw(
+        self,
+        type_name: str,
+        payload_idx: int,
+        modality: str,
```

Comment:
I would recommend making `modality` as enum instead of general string types. `Enum` is [supported](https://stackoverflow.com/questions/36932/how-can-i-represent-an-enum-in-python) up to python 2.4 with the library installed.